neard/src/runtime.rs

```diff
@@ -1395,7 +1395,8 @@ impl RuntimeAdapter for NightshadeRuntime {
     fn evm_chain_id(&self) -> u128 {
         match self.genesis_config.chain_id.as_str() {
             "mainnet" => MAINNET_EVM_CHAIN_ID,
-            _ => TEST_EVM_CHAIN_ID,
+            "testnet" => TESTNET_EVM_CHAIN_ID,
+            _ => BETANET_EVM_CHAIN_ID,
```

Comment:
Is it fine if local networks use the same evm chain id as betanet @artob?

Reply:
Yes, I think that's fine. Ideally, it would be configurable (as in, choose one of the betanet/testnet/mainnet chain IDs for your local development), but for now betanet is totally fine.